File: WebContent/page/app/model/entity/client/Recipient.js
/*
 * Project: MajekWMS
 * User   : kornicameister
 * Date   : 04.11.12
 * Time   : 01:26
 */

Ext.define('WMS.model.entity.client.Recipient', {
    extend: 'WMS.model.entity.Client',
    proxy : {
        type       : 'wms',
        url        : 'wms/agent/client',
        extraParams: {
            type: 'recipient'
        }
    }
});

File: javaSrc/wms/model/Recipient.java
package wms.model;

import javax.persistence.DiscriminatorValue;
import javax.persistence.Entity;

@Entity
@DiscriminatorValue(value = "recipient")
public class Recipient extends Client {
    private static final long serialVersionUID = -6775441124171801207L;

    public Recipient() {
        super();
    }

    @Override
    public String toString() {
        return "Recipient [toString()=" + super.toString() + "]";
    }

}